crates/bevy_core/src/time/fixed_timestep.rs

```diff
@@ -205,7 +208,9 @@ impl System for FixedTimestep {
             Box::new(Self::prepare_system.config(|c| c.0 = Some(self.state.clone())));
         self.internal_system.initialize(world);
         if let Some(ref label) = self.state.label {
-            let mut fixed_timesteps = world.get_resource_mut::<FixedTimesteps>().unwrap();
+            let mut fixed_timesteps = world
+                .get_resource_mut::<FixedTimesteps>()
+                .expect("Could not find `FixedTimesteps` in the `World`.");
```

Comment:
message should mention that `FixedTimesteps` is a resource, and that this resource is added by CorePlugin, DefaultPlugins or MinimalPlugins